File: exercises/RB101_small_problems/easy_3/searching_101.rb
def check_nums(arr, num)
  if arr.include?(num)
    puts "The number #{num} appears in #{arr}."
  else
    puts "The number #{num} does not appear in #{arr}."
  end
end

numbers = Array.new

puts "==> Enter the 1st number:"
numbers << gets.chomp.to_i
puts "==> Enter the 2nd number:"
numbers << gets.chomp.to_i
puts "==> Enter the 3rd number:"
numbers << gets.chomp.to_i
puts "==> Enter the 4th number:"
numbers << gets.chomp.to_i
puts "==> Enter the 5th number:"
numbers << gets.chomp.to_i
puts "==> Enter the last number:"
last = gets.chomp.to_i # check if last is in array

check_nums(numbers, last)

File: exercises/JS210_Small_Problems/Easy_3/searching101.js
let first = prompt('Enter the first number: ');
let second = prompt('Enter the second number: ');
let third = prompt('Enter the third number: ');
let fourth = prompt('Enter the fourth number: ');
let fifth = prompt('Enter the fifth number: ');
let last = prompt('Enter the last number: ');

const numbers = [first, second, third, fourth, fifth].map(s => Number(s));

if (numbers.includes(Number(last))) {
  console.log(`The number ${last} appears in [${numbers}]`);
} else {
  console.log(`The number ${last} does not appear in [${numbers}]`);
}